### PR TITLE
Spelling documentation

### PR DESCRIPTION
**Pull request by jsoref**
per https://github.com/mariadb-corporation/MaxScale/pull/282#issuecomment-1918566447

### CODE REVIEW

File: Documentation/Authenticators/Authentication-Modules.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ GRANT SHOW DATABASES ON *.* TO 'maxscale'@'maxscalehost';
 
 ## Limitations and troubleshooting
 
-When a client logs in to MaxScale, MaxScale sees the client's IP address. When
+When a client logs into MaxScale, MaxScale sees the client's IP address. When
```
Comment (jsoref) [inline Documentation/Authenticators/Authentication-Modules.md]: `into` changes tend to be controversial. happy to drop this or anything else.

File: Documentation/Authenticators/PAM-Authenticator.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ port=12345
 
 MaxScale uses the PAM authenticator plugin to authenticate users with *plugin*
 set to "pam" in the *mysql.user*-table. The PAM service name of a user is read
-from the *authetication_string*-column. The matching PAM service in the
+from the *authentication_string*-column. The matching PAM service in the
 operating system PAM config is used for authenticating the user. If the
-*authetication_string* for a user is empty, the fallback service "mysql" is
+*authentication_string* for a user is empty, the fallback service "mysql" is
```
Comment (jsoref) [inline Documentation/Authenticators/PAM-Authenticator.md:34]: All of these changes are now divorced from any corresponding code changes, so it's possible these are aspirational = breaking/broken.

File: Documentation/Protocols/NoSQL.md [hunk 3]
```diff
@@ -1608,7 +1608,7 @@ customData | document | Optional. Any arbitrary information.
 roles | array | Optional. The roles granted to the user. Note that the existing ones are _replaced_ and not amended with these roles.
 mechanisms | array | Optional. The specific SCRAM mechanisms for user credentials. Note that if a new `pwd` is provided, then the array can contain all supported SCRAM mechanisms. If a new `pwd` is not provided, then the array must be a subset of the existing mechanisms of the user.
 
-Changes to `customData` or `mechanisism` are made only to the local
+Changes to `customData` or `mechanisms` are made only to the local
```
Comment (jsoref) [inline Documentation/Protocols/NoSQL.md:1611]: likewise

File: Documentation/REST-API/Resources-Filter.md [hunk 4]
```diff
@@ -197,11 +197,11 @@ least the following fields.
 * `data.type`
   * Type of the object, must be `filters`
 
-* `data.atttributes.module`
+* `data.attributes.module`
   * The filter module to use
 
 All of the filter parameters should be defined at creation time in the
-`data.atttributes.parameters` object.
+`data.attributes.parameters` object.
```
Comment (jsoref) [inline Documentation/REST-API/Resources-Filter.md:204]: likewise

File: Documentation/Authenticators/Authentication-Modules.md (hunk 1 above)
Comment (markus456) [reply Documentation/Authenticators/Authentication-Modules.md]: I'm not a native speaker but to me this seems wrong as the act being described is _log in_ (to a device) not _log_ (fell a tree, write an entry to a journal). The same comment applies to all the _log in to_ changes in this commit.

File: Documentation/Authenticators/PAM-Authenticator.md (hunk 2 above)
Comment (markus456) [reply Documentation/Authenticators/PAM-Authenticator.md:34]: I'm fairly certain that this is just a typo in the documentation.

File: Documentation/Protocols/NoSQL.md [hunk 5]
```diff
@@ -672,7 +672,7 @@ which means that only the `test` database can be accessed and modified.
 
 Since `nosqlprotocol` is a regular protocol module used in a listener,
 the TLS/SSL support of listeners is available. Please see
-[TSLSSL encryption](Getting-Started/Configuration-Guide.md#tsl-encryption)
+[TLSSSL encryption](Getting-Started/Configuration-Guide.md#tls-encryption)
```
Comment (markus456) [inline Documentation/Protocols/NoSQL.md]: This technically should be `TLS/SSL` like is written in the line above it. Not necessarily required but if you want to, I'd say you can correct this as well.

File: Documentation/Protocols/NoSQL.md (hunk 3 above)
Comment (markus456) [reply Documentation/Protocols/NoSQL.md:1611]: I can confirm that this is just a typo.

File: Documentation/REST-API/Resources-Filter.md (hunk 4 above)
Comment (markus456) [reply Documentation/REST-API/Resources-Filter.md:204]: Also just a typo in the documentation.

File: Documentation/Routers/ReadWriteSplit.md [hunk 6]
```diff
@@ -1058,7 +1058,7 @@ secondary servers. Readwritesplit is an exception to this rule. The following
 rules govern how readwritesplit behaves with servers that have different ranks.
 
 * Sessions will use the current primary server as long as possible. This means
-  that sessions with a secondary primary will not use the primary primary as long
+  that sessions with a secondary primary will not use the primary as long
```
Comment (markus456) [inline Documentation/Routers/ReadWriteSplit.md]: The duplication in this case, while horribly confusing, was intentional. You can replace the `primary primary` with `main primary`.

**Review by markus456 — CHANGES_REQUESTED** [review]

File: Documentation/Authenticators/Authentication-Modules.md (hunk 1 above)
Comment (jhnwkmn) [reply Documentation/Authenticators/Authentication-Modules.md]: Yes, should be "log in to" and not "log into".

_The proper grammar is "log in to." The phrase "log in" is a verb phrase that means to gain access to a computer system by entering a username and password. The preposition "to" follows the verb "log in" to indicate the thing that you are logging in to, such as a website or a computer program._

**Comment by markus456** [comment]
As per the [Contribution policy](https://github.com/mariadb-corporation/MaxScale/wiki/Contributing) of MaxScale, please indicate the license under which this change is submitted. If you wish to submit the changes under the BSD-new license, you can respond with the following comment:

> I am contributing the new code of the whole pull request, including one or
> several files that are either new files or modified ones, under the BSD-new
> license.

**Comment by jsoref** [comment]
I am contributing the new code of the whole pull request, including one or
several files that are either new files or modified ones, under the BSD-new
license.

**Review by markus456 — APPROVED** [review]

**Review by jhnwkmn — APPROVED** [review]

**Comment by markus456** [comment]
Sorry for not merging this in yet (vacations etc.) but could you fix those few conflicts that have appeared? Once the merge is free of conflicts, I'll squash that into a single commit and merge it.

**Comment by markus456** [comment]
Thank you very much!

**Pull request merged**